portfolio/src/main/webapp/jsPractice.js

```diff
@@ -0,0 +1,8 @@
+/** Fetches the current date from the server and adds it to the page. */

```

Comment:
Is this code called from anywhere? It looks correct but I don't see where it's used. I'd suggest either moving it into script.js or adding the script in index.html so you can actually test it. Let me know if you run into issues with this or have other questions!